packages/api/src/common/dapps/flux/getFluxCdps.test.ts

```diff
@@ -0,0 +1,82 @@
+import { Effect, Layer } from 'effect';
+import { describe, it } from 'vitest';
+import { EntityNonFungibleDataService } from '../../gateway/entityNonFungiblesData';
+import { EntityNonFungiblesPageService } from '../../gateway/entityNonFungiblesPage';
+import { GatewayApiClientLive } from '../../gateway/gatewayApiClient';
+import { GetEntityDetailsService } from '../../gateway/getEntityDetails';
+import { GetLedgerStateService } from '../../gateway/getLedgerState';
+import { GetNftResourceManagersService } from '../../gateway/getNftResourceManagers';
+import { GetNonFungibleBalanceService } from '../../gateway/getNonFungibleBalance';
+import { GetNonFungibleIdsService } from '../../gateway/getNonFungibleIds';
+import { KeyValueStoreDataService } from '../../gateway/keyValueStoreData';
+import { KeyValueStoreKeysService } from '../../gateway/keyValueStoreKeys';
+import { GetFluxCdpsService } from './getFluxCdps';
+
+const fullLayer = GetFluxCdpsService.Default.pipe(
```

Comment:
you don't need to provide dependencies here as they are already provided in the service